draft-ietf-taps-interface.md

```diff
@@ -520,31 +538,16 @@ for Protocol Specific Properties and MUST NOT be used for vendor or implementati
 
 ### Transport Property Types {#property-types}
 
-Transport Properties can have one of a set of data types:
-
-- Boolean: can take the values `true` and `false`; representation is
-  implementation-dependent.
-- Integer: can take positive or negative numeric integer values; range and
-  representation is implementation-dependent.
-- Numeric: can take positive or negative numeric values; range and
-  representation is implementation-dependent.
-- Enumeration: can take one value of a finite set of values, dependent on the
-  property itself. The representation is implementation dependent.
-- Preference: can take one of five values (Prohibit, Avoid, Ignore, Prefer,
-  Require) for the level of preference of a given property during protocol
-  selection; see {{selection-props}}. When querying, a Preference is of
-  type Boolean, with `true` indicating that the Selection Property
-  has been applied.
-- Tuple: An ordered grouping of multiple value types.
-  In this this document, it is written as a list in brackets, e.g., ```(Enumeration, Preference)``` 
-  The composition of types and their order depends on the property and is fixed for the property.
-  The actual representation is implementation-dependent.
-- Collection: An unordered grouping of one or more values of the same type.
-  The actual representation, e.g. as a set or an array, is implementation-dependent.
+Transport Properties each have a type, which can be:
 
-For types Integer and Numeric, special values can be defined
-per property; it is up to implementations how these special values are
-represented (e.g., by using -1 for an otherwise non-negative value).
+- One of the basic types described in {{notation}}; or
+- Preference, which on Preconnection is an Enumeration with six possible
+  values: Prohibit, Avoid, Ignore, Prefer, Require, or Default. Each of the
+  first five denotes a level of preference of a given property during protocol
+  selection, while Default indicates deference to an implementation's preference.
+  (See {{selection-props}}.) When querying the properties of a Connection
+  following protocol selection, a Preference is of type Boolean, with `true`
+  indicating that the Selection Property applies to the chosen transport.
```

Comment:
I think this can be a bit confusing. It might be better to just keep the first two sentences here and move the last one to after the table (line 732).